examples/cpp/dds/SecureHelloWorldExample/HelloWorldPublisher.cpp

```diff
@@ -18,9 +18,12 @@
  */
 
 #include "HelloWorldPublisher.h"
+
 #include <fastdds/dds/domain/DomainParticipantFactory.hpp>
 #include <fastdds/dds/subscriber/SampleInfo.hpp>
 
+#include "HelloWorldPubSubTypes.hpp"
+
 using namespace eprosima::fastdds::dds;
 
 HelloWorldPublisher::HelloWorldPublisher()
```

Comment:
Please, remove the old `SecureHelloworldExample` and also from the corresponding `CMakeLists.txt`